index.html

```diff
@@ -1736,24 +1718,20 @@
       </section>
       <section>
         <h3>
-          Interface <a><code>PresentationReceiver</code></a>
+          Interface <a><code>PresentationConnectionList</code></a>
         </h3>
         <pre class="idl">
-          interface PresentationReceiver : EventTarget {
-            Promise&lt;PresentationConnection&gt; getConnection();
-            Promise&lt;sequence&lt;PresentationConnection&gt;&gt; getConnections();
+          interface PresentationConnectionList : EventTarget {
+            readonly attribute FrozenArray&lt;PresentationConnection&gt; connections;
             attribute EventHandler onconnectionavailable;
           };
 
-        
+
 </pre>
         <p>
-          The <a>PresentationReceiver</a> object is available to a <a>receiving
-          browsing context</a> in order to access the <a data-lt=
-          "controlling browsing context">controlling browsing context</a> and
-          communicate with it. The <a><code>PresentationReceiver</code></a>
-          object MUST be implemented in a <a>receiving browsing context</a>
-          provided by a <a>receiving user agent</a>.
+          The <a for="PresentationConnectionList">connections</a> attribute
+          MUST return a the non-terminated <a>set of presentations</a>
```

Comment:
Typo done.

"non-terminated" indeed means state being not terminated. When we discussed this in the issue, we decided to include "closed" connections because they might re-open.